chaos/Readme.md

```diff
@@ -0,0 +1,616 @@
+6 records sent, 0.8 records/sec (0.00 MB/sec), 1198.5 ms avg latency, 6632.0 ms max latency.
+org.apache.kafka.common.errors.InvalidRequiredAcksException: Produce request specified an invalid value for required acks.
+[2022-11-16 17:00:42,491] WARN [Producer clientId=perf-producer-client] Got error produce response with correlation id 8 on topic-partition conduktor_topic-0, retrying (2147483646 attempts left). Error: NOT_ENOUGH_REPLICAS (org.apache.kafka.clients.producer.internals.Sender)
+[2022-11-16 17:00:42,492] WARN [Producer clientId=perf-producer-client] Got error produce response with correlation id 9 on topic-partition conduktor_topic-0, retrying (2147483646 attempts left). Error: NOT_ENOUGH_REPLICAS (org.apache.kafka.clients.producer.internals.Sender)
+100 records sent, 9.999000 records/sec (0.00 MB/sec), 2454.80 ms avg latency, 6852.00 ms max latency, 2046 ms 50th, 6560 ms 95th, 6852 ms 99th, 6852 ms 99.9th.
+```
+
+Note the `NOT_ENOUGH_REPLICAS` errors.
+
+### Step 6: Reset
+
+To stop chaos injection run the below:
+
+```bash
+docker-compose exec kafka-client curl \
+    --silent \
+    --request DELETE "conduktor-proxy:8888/tenant/1-1/user/test@conduktor.io/feature/broken-broker/apiKeys/PRODUCE/direction/REQUEST"
+```
+
+### Step 7: Run with no Chaos
+
+To verify, let's run the produce test again to confirm there are no errors
+
+```bash
+docker-compose exec kafka-client kafka-producer-perf-test \
+  --producer.config /clientConfig/proxy.properties \
+  --record-size 100 \
+  --throughput 10 \
+  --num-records 100 \
+  --topic conduktor_topic
+```
+
+This should produce output similar to the following:
+
+```bash
+52 records sent, 10.3 records/sec (0.00 MB/sec), 16.0 ms avg latency, 388.0 ms max latency.
+100 records sent, 10.001000 records/sec (0.00 MB/sec), 10.69 ms avg latency, 388.00 ms max latency, 5 ms 50th, 43 ms 95th, 388 ms 99th, 388 ms 99.9th.
+```
+
+### <a name="duplicateWrites"></a> Step 8: Duplicate Writes
+
+Conduktor Proxy exposes a REST API to configure the chaos features.
+
+The command below will instruct Conduktor Proxy to inject duplicate records on produce requests.
+
+```bash
+docker-compose exec kafka-client \
+  kafka-topics \
+    --bootstrap-server conduktor-proxy:6969 \
+    --command-config /clientConfig/proxy.properties \
+    --create --if-not-exists \
+    --topic conduktor_topic_duplicate
+```
+
+```bash
+docker-compose exec kafka-client curl \
+    --silent \
+    --request POST "conduktor-proxy:8888/tenant/1-1/user/test@conduktor.io/feature/duplicate-resource" \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+        "config": { 
+          "topics": ["conduktor_topic_duplicate"], 
+          "duration": 1, 
+          "rateInPercent": 100, 
+          "quietPeriod": 1, 
+          "timeUnit": "MINUTES" 
+        },
+        "direction": "REQUEST",
+        "apiKeys": "PRODUCE"
```

Comment:
Some interceptors will default apikey and direction. will not need to pass them in when configuring.
[InterceptorType](https://github.com/conduktor/conduktor-proxy/blob/main/proxy/src/main/java/io/conduktor/proxy/model/InterceptorType.java)

Reply:
That's cool, I like the explicit definition for the demo as it gives users a really clear idea of what is being applied.